src/timeseriesflattener/feature_spec_objects.py

```diff
@@ -89,7 +91,12 @@ def resolve_values_df(data: dict[str, Any]):
 
 
 class BaseModel(PydanticBaseModel):
-    """."""
+    """Modified Pydantic BaseModel to allow arbitrary
+    types and disallow attributes not in the class."""
```

Comment:
Allegedly, Sphinx already pulls parent docstrings. When merging, can you check if that's the case in the docs? If so, we'll have to disable it in Sphinx.

https://stackoverflow.com/questions/2025562/inherit-docstrings-in-python-class-inheritance

Reply:
Will do!